sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -121,3 +128,17 @@ export class AESTStudentFileDTO extends StudentUploadFileDTO {
 export class StudentFileMetadataDTO {
   applicationNumber?: string;
 }
+
+export class StudentInfo {
```

Comment:
I believe that at this level the class should be named DTO, right?

Reply:
I actually coped this interface from a different file and pasted it here. since there is a story to reorganize student API, i didn't renamed the DTO